src/scenes/game-scene.ts

```diff
@@ -46,29 +48,61 @@ const levelConfig = {
   ],
 };
 
+const soundConfig = [
+  {
+    key: 'punch',
+    filePath: 'audio/sfx/punch.wav',
```

Comment:
im not 100% sure, but i think these file paths have to start with `./` or they dont work after deployment